modules/gdscript/gdscript_parser.cpp

```diff
@@ -4442,7 +4442,8 @@ void GDScriptParser::_parse_class(ClassNode *p_class) {
 				rpc_mode = MultiplayerAPI::RPC_MODE_MASTER;
 				continue;
 			} break;
-			case GDScriptTokenizer::TK_PR_SLAVE: {
+			case GDScriptTokenizer::TK_PR_SLAVE:
```

Comment:
Should we add something like:
```cpp
ERR_EXPLAIN("GDScript's 'slave' keyword is deprecated and will be removed in a future release, please replace its uses by 'puppet'.");
WARN_DEPRECATED
```
and fall through to the next case?

Reply:
What about using the beautiful and brand new @vnen warning messages?
![deprecated](https://user-images.githubusercontent.com/1687918/45579672-fed1e900-b889-11e8-93ae-6ed4724b3fb0.png)

Reply:
:heart: